acl_loader/main.py

```diff
@@ -579,6 +579,9 @@ def show_rule(self, table_name, rule_id):
 
             matches.sort()
 
+            if len(matches) == 0:
+                matches.append("")
```

Comment:
If no matches are found, is it better to leave the column empty (this solution), or print `N/A` instead?

Reply:
`N/A` would be good

Reply:
example output:
```
admin@switch:~$ show acl rule 
Table    Rule          Priority    Action    Match
-------  ------------  ----------  --------  --------------------
TESTACL  RULE_1        9999        FORWARD   DST_IPV6: 2002::1/64
                                             SRC_IPV6: 2001::1/64
TESTACL  DEFAULT_RULE  1           DROP      N/A
```